packages/cli/src/ui/hooks/slashCommandProcessor.ts

```diff
@@ -55,6 +55,10 @@ export const useSlashCommandProcessor = (
 ) => {
   const session = useSessionStats();
   const [commands, setCommands] = useState<readonly SlashCommand[]>([]);
+  const [commandEpoch, setCommandEpoch] = useState(0);
```

Comment:
this is a bit of an antipattern. rather than incrementing the epoc to force a refresh can we get the configuration you care about stored so that it properly flows through react? happy to brainstorm some ways you might be able to do this that might be better aligned with react. if there isn't an easy way we can use this hack to unblock you.